Automattic/wp-calypso
# Pricing Page Rework: Add short descriptions into the product cards

This PR updates the new Pricing Page Rework product cards to display new, shorter product descriptions as per the design.

#### Proposed Changes

* Define short descriptions for all Jetpack "products" in `@automattic/calypso-products`
* Add a `shortDescription` property to the `selectorProduct` object.
* Display the short description in the pricing page displayed products.

Note: Short description for Jetpack Social product is not included in this PR because Jetapck Social product data does not exist yet. We'll need to add Jetpack Social product data in a follow-up PR.

#### Testing Instructions

* click on Jetpack Cloud live link below
    * Goto `/pricing?flags=jetpack/pricing-page-rework-v1`
* or boot up this PR 
    * Run `git fetch && git checkout add/new-product-short-descriptions`
    * Run `yarn start-jetpack-cloud`
    * Goto http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1
* View all the product cards and confirm that the product descriptions match the "Products Section" in this doc: https://docs.google.com/document/d/1vvzHYMsaV0EMlF4BLbvVa-dgq--9iS-I59gdic3ZjIA/edit#heading=h.nb3gcjg23dow

#### Screenshots
**BEFORE:**

![Markup 2022-08-30 at 19 51 58](https://user-images.githubusercontent.com/11078128/187563460-0b147876-7d25-4adc-9a26-bb4f5655add6.png)

-------------------------------

**AFTER:** 
Product cards **with new short descriptions**:

![Markup 2022-08-30 at 17 34 12](https://user-images.githubusercontent.com/11078128/187547769-cc0e4350-c3e9-460b-bbbf-b88eeb7e0a02.png)


#### Pre-merge Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202886227656205
